vcxagency-node/src/app.js

```diff
@@ -63,26 +63,14 @@ async function wireUpApplication ({
     }
     const redisClientSubscriber = redis.createClient(redisUrl)
     const redisClientRw = redis.createClient(redisUrl)
-    redisClientRw.on('error', function (err) {
-      logger.error(`Redis rw-client encountered error: ${err}`)
-    })
-    redisClientSubscriber.on('error', function (err) {
-      logger.error(`Redis subscription-client encountered error: ${err}`)
-    })
-    redisClientRw.on('connect', function () {
-      logger.info('Redis rw-client connected.')
-    })
-    redisClientSubscriber.on('connect', function () {
-      logger.info('Redis subscription-client connected.')
-    })
     serviceNewMessages = createServiceNewMessages(redisClientSubscriber, redisClientRw)
   } else {
     throw Error(`Unknown agency type ${agencyType}`)
   }
 
   const serviceIndyWallets = await createServiceIndyWallets(walletStorageType, walletStorageConfig, walletStorageCredentials)
   const { user, password, host, port, database } = appStorageConfig
-  await waitUntilConnectsToPostgres(appStorageConfig, 1, 2000)
+  await waitUntilConnectsToPostgres(appStorageConfig, 1, 10000)
```

Comment:
Hmm, I wonder why I originally set it to `1, 2000`. Maybe rather something like `(5, 2000)`?